api/src/paths/project/{projectId}/attachments/{attachmentId}/metadata/get.ts

```diff
@@ -154,59 +181,32 @@ export function getProjectReportMetaData(): RequestHandler {
     const connection = getDBConnection(req['keycloak_token']);
 
     try {
-      const getProjectReportAttachmentSQLStatement = queries.project.getProjectReportAttachmentSQL(
-        Number(req.params.projectId),
-        Number(req.params.attachmentId)
-      );
+      await connection.open();
+
+      const attachmentService = new AttachmentService(connection);
 
-      const getProjectReportAuthorsSQLStatement = queries.project.getProjectReportAuthorsSQL(
+      const projectReportAttachment = await attachmentService.getProjectReportAttachment(
+        Number(req.params.projectId),
         Number(req.params.attachmentId)
       );
 
-      if (!getProjectReportAttachmentSQLStatement || !getProjectReportAuthorsSQLStatement) {
-        throw new HTTP400('Failed to build metadata SQLStatement');
-      }
-
-      await connection.open();
+      const projectReportAuthors = await attachmentService.getProjectAttachmentAuthors(Number(req.params.attachmentId));
 
-      const reportMetaData = await connection.query(
-        getProjectReportAttachmentSQLStatement.text,
-        getProjectReportAttachmentSQLStatement.values
-      );
-
-      const reportAuthorsData = await connection.query(
-        getProjectReportAuthorsSQLStatement.text,
-        getProjectReportAuthorsSQLStatement.values
+      const projectReportSecurity = await attachmentService.getProjectAttachmentSecurityRules(
```

Comment:
At some point we may want to do a pass for consistentcy re: "rules" and "reasons" verbiage